portfolio/src/main/java/com/google/sps/servlets/ParameterFetcher.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+import javax.servlet.http.HttpServletRequest;
+
+/* purpose: fetches parameters from HTML elements that have sent browser requests **/
+class ParameterFetcher {
+    private ParameterFetcher() {};
+
+    /* wrapper for the servlet request method, with safety default value*/
+    public static String getParameter(HttpServletRequest request, String name) {
```

Comment:
It's weird that it's a ParameterFetcher but the method is called "get" instead of "fetch"

Reply:
Changed to ParameterGetter